TrackerConditions/src/StrawElectronicsMaker.cc

```diff
@@ -116,7 +120,7 @@ namespace mu2e {
 	// correct for sampleRate so that calculateResponse peak is independent of it
 	// this combined with pC_per_uA_ns is the unit transform from pC to uA
 	wPoints[ai]._currentPulse[i] /= sampleRate * pC_per_uA_ns;
-	wPoints[ai]._currentPulse[i] /= 4.615; // normalization for 1/(t+t0)
```

Comment:
Is this a number we expect to ever change?  Can it be calculated from anything else?

Reply:
This has been replaced with the new variable integral_normalization that is calculated from the waveform